app/src/main/java/org/projectbuendia/client/models/Obs.java

```diff
@@ -28,62 +27,77 @@
 import static com.google.common.base.Preconditions.checkNotNull;
 
 /** A simple bean class representing an observation with localized names and values. */
-public final class ObsValue implements Comparable<ObsValue> {
-    public static final Comparator<ObsValue> BY_OBS_TIME = new Comparator<ObsValue>() {
-        @Override public int compare(ObsValue left, ObsValue right) {
-            return left.obsTime.compareTo(right.obsTime);
-        }
-    };
-
+// TODO: Make ObsPoint a member of Obs; change the structure of Obs to be simply:
+// { final @Nonnull String uuid; String name; final @Nonnull ObsPoint point; } then delete
+// getObsPoint(), getObsValue(), compareTo(), getTypeOrdering(), getCodedValueOrdering().
```

Comment:
I see that ObsValue on the other hand does have the logic to compare coded values.

Should ObsPoint compare by time (as it does now) or by value (which would use the ObsValue compareTo() method)?